libp2p/autotls/service.nim

```diff
@@ -135,12 +156,14 @@ when defined(libp2p_autotls_support):
 
   method issueCertificate(
       self: AutotlsService
-  ) {.
+  ): Future[bool] {.
```

Comment:
hmm with this change code needs to handle errors and check result when doing `issueCertificate`? return value kind of looks  redundant here. what was reasoning for this change?